pkg/deploy/registry/deploylog/rest.go

```diff
@@ -138,7 +138,12 @@ func (r *REST) Get(ctx kapi.Context, name string, opts runtime.Object) (runtime.
 	// Setup url of the deployer pod
 	deployPodName := deployutil.DeployerPodNameForDeployment(target.Name)
 	logOpts := &kapi.PodLogOptions{
-		Follow: deployLogOpts.Follow,
+		Follow:       deployLogOpts.Follow,
```

Comment:
This is the other spot that would benefit from the helper to create a PodLogOptions from the DeployLogOptions.